corehq/apps/users/views/mobile/users.py

```diff
@@ -756,11 +763,61 @@ def create_mobile_worker(self, in_data):
             phone_number = self.new_mobile_worker_form.cleaned_data['phone_number']
             couch_user.set_default_phone_number(phone_number)
             send_account_confirmation_sms_if_necessary(couch_user)
+
+        data = self.get_plan_and_user_count_by_domain(self.domain)
+        self.check_and_send_limit_email(self.domain, data['plan_limit'], data['user_count'])
         return {
             'success': True,
             'user_id': couch_user.userID,
         }
 
+    @staticmethod
+    def get_plan_and_user_count_by_domain(domain):
+        subscription = Subscription.get_active_subscription_by_domain(domain)
+        plan_version = subscription.plan_version if subscription else DefaultProductPlan.get_default_plan_version()
+        for rate in plan_version.feature_rates.all():
+            if rate.feature.feature_type == 'User':
+                user_rate = rate
+                break
+        return {
+            'plan_limit': user_rate.monthly_limit,
+            'user_count': FeatureUsageCalculator(user_rate, domain).get_usage()
+        }
```

Comment:
I notice that all usages of this essentially boil down to:
```
data = get_plan_and_user_count_by_domain(...)`
do_something(data['plan_limit'], data['user_count'])
```

I appreciate the readability of an object with specific keys, but I feel like that readability is mostly offset by kind of pushing the user into using a generic `data` variable. Javascript has a clean solution to this, as you can just write:
`const {plan_limit, user_count} = get_plan_and_user_count_by_domain(...)`. There is some discussion of the ways we could get Javascript-like functionality [here](https://stackoverflow.com/questions/54785148/destructuring-dicts-and-objects-in-python), but I wonder if it's just more Pythonic to make the function itself slightly less readable by returning a tuple instead of a dict, i.e.:
` return (user_rate.monthly_limit, FeatureUsageCalculator(user_rate, domain).get_usage()`
so that it enables just doing:
` plan_limit, user_count = get_plan_and_user_count_by_domain(...)`